src/test/java/com/sumologic/log4j/SumoLogicAppenderTest.java

```diff
@@ -105,4 +105,10 @@ public void testMessagesWithoutMetadata() throws Exception {
             assertEquals("log4j2-appender", request.getHeaders().getFirst("X-Sumo-Client"));
         }
     }
+
+    @Test
+    public void setupAppenderWithoutRequiredFields() {
```

Comment:
Not sure if there is any value in such a test. wdyt?

Reply:
Doesn't hurt, but maybe I am missing something - see the other comment wrt `null`.